ldapauthenticator/ldapauthenticator.py

```diff
@@ -94,6 +95,29 @@ def _observe_use_ssl(self, change):
         """,
     )
 
+    tls_kvargs = Dict(
```

Comment:
keyword arguments are ~always called `kwargs`. Or are these something different?

```suggestion
    tls_kwargs = Dict(
```

Reply:
Aaaaaaaaaah thanks for catching this!! I felt something was off but I didn't figure out what.